src/Intro/Intro.tsx

```diff
@@ -46,70 +50,62 @@ const styles: any = (theme: any) => ({
   },
 });
 
-function withStylesTyped(myStyles: any) {
-  return (Component: any) => {
-    return withStyles(myStyles)(Component) as any;
-  };
-}
+// FIXME: any type
+const intersperse = (arr: JSX.Element[], sep: string) =>
+  arr
+    .reduce((acc: JSX.Element[], prev: any) => [...acc, prev, sep], [])
+    .slice(0, -1);
```

Comment:
Why slice the last one?

Reply:
We would have to ask @Tenchi2xh I just moved this function